mission/functions/core/ui/taskroster/fn_tr_overview_init.sqf

```diff
@@ -71,16 +71,18 @@ if(!isNull _task)then
 };
 
 //set Zone names and setup the Ctrl with the proper Filter
+private _activeZoneNames = missionNamespace getVariable ["mf_g_dir_activeZoneNames", []];
```

Comment:
Related to rework in mission/functions/systems/director/fn_director_init.sqf

will want to test this.